README.md

```diff
@@ -235,6 +235,24 @@ docker-compose logs -f sidekiq
 podman logs -f compliance-backend_sidekiq_1
 ```
 
+### Building the image
+
+This project uses `build_deploy.sh` to build and push this image to a remote registry.
+You must set the required variables to provide authentication to the required registries.
+You must set at least the Red Hat Registry credentials to be able to pull the base image.
+
+Example:
+```
+LOCAL_BUILD=true RH_REGISTRY_USER=guybrush RH_REGISTRY_TOKEN=M0nk3y ./build_deploy
```

Comment:
```suggestion
LOCAL_BUILD=true RH_REGISTRY_USER=guybrush RH_REGISTRY_TOKEN=M0nk3y ./build_deploy.sh
```

Reply:
thanks ! updated! (sorry I didn't see the suggestion mode :D )